test/options.cpp

```diff
@@ -21,6 +21,8 @@ class Argv {
       m_args.push_back(std::move(ptr));
       m_argv.get()[i] = m_args.back().get();
 
+      m_argv_str += std::string(m_argv.get()[i]) + std::string(" ");
```

Comment:
Nearly all tests made for the `(argc, argv)` pair could be used for this single string code, unless we later on decide to design a separate set of tests for single string code and put them into a different cpp file. I would suggest we put the tests for `(argc, argv)` pair and single string together, so that whenever a new test is added, both functions can be tested.

Reply:
I agree that this is not necessary. You really only need to test that the string splitting works.

Reply:
Oh, sorry, I take it back! I didn't notice this is was in one of the tests... :-(